docker-compose.yml

```diff
@@ -18,7 +17,7 @@ services:
       - mysql
     restart: unless-stopped
   mysql:
-    image: mysql-local:1.0.0
+    image: mysql-local:${MYSQL_VERSION:-1.0.0}
```

Comment:
Ensure that the `MYSQL_VERSION` environment variable is set and that the image `mysql-local` with the specified version exists in your Docker registry. If not, the default version `1.0.0` will be used, which may not exist.